src/glossary.rst

```diff
@@ -5890,8 +5891,7 @@ rustc
 ^^^^^
 
 :dp:`fls_zdgbeixirjfm`
-Rust-c (or :dt:`rustc`) is a compiler that implements the Ferrocene Language
-Specification.
+Rust-c (or :dt:`rustc`) is a compiler that implements the FLS.
```

Comment:
```suggestion
:dt:`rustc` is a compiler that implements the FLS.
```

Since we're touching this line, we might as well remove this "Rust-c" thing.  We never call it that.